SyntheticsRunTestsTask/package.json

```diff
@@ -25,7 +25,7 @@
     "node": "10.21.0"
   },
   "dependencies": {
-    "@datadog/datadog-ci": "^1.12.0",
+    "@datadog/datadog-ci": "^1.16.0",
```

Comment:
The `^` seems "dangerous" as we tend to publish breaking changes with minor versions of datadog-ci (e.g. `failOnMissingTests` flag, `parseConfigFile` that is deprecated and will probably be removed in a minor version).

Shouldn't we use `~` instead?

Reply:
I didn't want to change it in this PR, but yes I agree we should either pin the version or use `~`.

Reply:
After discussion, we decided to keep the `^`.